code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -270,9 +273,28 @@
 		return
 
 	var/module_type = robot_modules[modtype]
-	new module_type(src)
-
-//	hands.icon_state = lowertext(modtype)
+	var/obj/item/weapon/robot_module/RM = new module_type() //Spawn a dummy module to read values from
+
+	switch(alert(src, "[RM.desc] \n \n\
```

Comment:
Oh my this is hard to read.

Reply:
I put those linebreaks in there to make it easier to read!